README.md

```diff
@@ -255,6 +255,11 @@ $permissions->hasFlags(Permissions::Write); // False
 Hence, using `FlagBag::getValue()` you can get an encoded value for any combination of flags from your enum, 
 and use it for storage or communication between your processes.
 
+You can initiate a "full" bag with all its flags set by calling the `fromAll` static method:
+```php
+$permissions = FlagBag::fromAll(Permissions::class); // Returns an empty bag
```

Comment:
Let's just add this example with the others above. For instance, after:

```php
$permissions = FlagBag::from(Permissions::Execute, Permissions::Write, Permissions::Read);
// same as:
$permissions = new FlagBag(Permissions::class, 7); 
// where 7 is the "encoded" bits value for:
Permissions::Execute->value | Permissions::Write->value | Permissions::Read->value // 7
```

add:

```php
// or, you can also initiate a bag with all its flags using:
$permissions = FlagBag::fromAll(Permissions::class);
```